scripts/functions.R

```diff
@@ -1,39 +1,143 @@
-##funciton ot find a string in your directory from https://stackoverflow.com/questions/45502010/is-there-an-r-version-of-rstudios-find-in-files
-
-fif <- function(what, where=".", in_files="\\.[Rr]$", recursive = TRUE,
-                ignore.case = TRUE) {
-  fils <- list.files(path = where, pattern = in_files, recursive = recursive)
-  found <- FALSE
-  file_cmd <- Sys.which("file")
-  for (fil in fils) {
-
-    if (nchar(file_cmd) > 0) {
-      ftype <- system2(file_cmd, fil, TRUE)
-      if (!grepl("text", ftype)[1]) next
-    }
-    contents <- readLines(fil)
-    res <- grepl(what, contents, ignore.case = ignore.case)
-    res <- which(res)
-    if (length(res) > 0) {
-      found <-  TRUE
-      cat(sprintf("%s\n", fil), sep="")
-      cat(sprintf(" % 4s: %s\n", res, contents[res]), sep="")
-    }
-  }
-  if (!found) message("(No results found)")
+# build reporting table for minnow trap surveys
+tab_fish_mt <- function(sit = my_site){
+    hab_fish_indiv %>%
+    filter(site == sit & sampling_method == 'minnow trapping' & !is.na(life_stage)) %>%
+    group_by(site, location, species_code, life_stage) %>%
+    summarise(Number = n()) %>%
+    ungroup() %>%
+    select(Location = location,
+           Species = species_code,
+           Stage = life_stage,
+           Number) %>%
+    pivot_wider(names_from = Stage,
+                values_from = Number) %>%
+    mutate(Location = case_when(Location == 'us' ~ 'Upstream',
+                                T ~ 'Downstream')) %>%
+    arrange(Species) %>%
+    replace(is.na(.), 0)
 }
 
+# take habitat confirmation xls and make a csv with the alias_local_name pasted to the comments
+# so people can see which comments are linked to which site
+fpr_hab_alias_to_comments <- function(target = target_dir){
```

Comment:
use `tfpr_` or something else divergent from `fpr`as prefix so we won't ever get conflicts